concent_api/core/views.py

```diff
@@ -421,8 +421,8 @@ def handle_send_force_subtask_results(request, client_message: message.concents.
                 reason      = message.concents.ServiceRefused.REASON.TooSmallProviderDeposit,
             )
 
-    client_message_send_to_soon = client_message.ack_report_computed_task.timestamp + settings.CONCENT_MESSAGING_TIME
-    client_message_send_to_late = client_message.ack_report_computed_task.timestamp + settings.CONCENT_MESSAGING_TIME + settings.FORCE_ACCEPTANCE_TIME
+    client_message_send_to_soon = client_message.ack_report_computed_task.timestamp + settings.SUBTASK_VERIFICATION_TIME
+    client_message_send_to_late = client_message.ack_report_computed_task.timestamp + settings.SUBTASK_VERIFICATION_TIME + settings.FORCE_ACCEPTANCE_TIME
```

Comment:
I have not noticed this earlier but you have an error in your names. It should be `..._too_soon` and `..._too_late`.

Reply:
ups, my mistake.

Reply:
:+1:  I have fixed it.